### DISCUSSION TITLE
Inconsistent Timeframe Representation in Different Chart Intervals

### DISCUSSION THREAD

**Discussion by NijanJ** [highfestiva/finplot#435] [Q&A]
I have encountered an issue with the representation of timeframes in different chart intervals in my trading application. The problem arises when switching between a 1-minute timeframe and a 15-minute timeframe.

Expected Behavior:
In the Indian market, the trading day starts at 9:30 AM and ends at 15:30 PM. When using a 1-minute timeframe, the timestamps are correctly aligned with the trading hours.

Observed Behavior:
However, when I switch to a 15-minute timeframe, each trading day starts at 15:00 PM and ends at 21:00 PM. This inconsistency in the representation of timeframes is causing confusion and affecting the accuracy of my analysis.

**Comment by highfestiva** [comment]
This I would guess has to do with your input data. If you print the `DataFrame` you're plotting, you'll see that the time differs. Also, the display time of finplot defaults to UTC. See [here](https://github.com/highfestiva/finplot/wiki/Snippets#time-zone) for more info.

**Comment by NijanJ** [comment]
Thank you for your prompt response and assistance with my question. Your help is greatly appreciated! I will follow your suggestion and implement the changes accordingly. If I have any further questions, I'll reach out to you. Thanks again!
